models/tag.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, Sequelize) => {
+  class Tag extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+      this.hasMany(models.Sprint, {
+        foreignKey: 'tagId',
+        targetKey: 'id'
+      });
+      this.belongsTo(models.Workspace, {
+        foreignKey: 'workspaceId',
+        targetKey: 'id'
+      });
```

Comment:
tags should belong to Task not workspace

Reply:
ok